src/Eurofurence.App.Server.Web/Controllers/ArtistsAlleyController.cs

```diff
@@ -20,13 +21,60 @@ public class ArtistsAlleyController : BaseController
     {
         private readonly ITableRegistrationService _tableRegistrationService;
         private readonly IImageService _imageService;
+        private readonly IArtistAlleyUserPenaltyService _artistAlleyUserPenaltyService;
 
-        public ArtistsAlleyController(ITableRegistrationService tableRegistrationService, IImageService imageService)
+        private const string ArtistAlleyDisabledFilePath = ".ArtistAlleyDisabled";
+
+        public ArtistsAlleyController(ITableRegistrationService tableRegistrationService, IImageService imageService, IArtistAlleyUserPenaltyService artistAlleyUserPenaltyService)
         {
             _tableRegistrationService = tableRegistrationService;
             _imageService = imageService;
+            _artistAlleyUserPenaltyService = artistAlleyUserPenaltyService;
+        }
+
+        /// <summary>
+        /// Returns the current system status of the artist alley
+        /// </summary>
+        /// <returns>The current status of the artist alley system</returns>
+        [HttpGet("SystemStatus")]
+        [ProducesResponseType(typeof(ArtistAlleySystemStatus), 200)]
+        [Authorize]
+        public ArtistAlleySystemStatus GetArtistAlleyStatus()
+        {
+            if (System.IO.File.Exists(ArtistAlleyDisabledFilePath))
+            {
+                return ArtistAlleySystemStatus.REG_CLOSED;
+            }
+            return ArtistAlleySystemStatus.OPEN;
         }
 
+        /// <summary>
+        /// Changes the system status of the artist alley system
+        /// </summary>
+        /// <param name="status">To new status of the artist alley system</param>
+        /// <returns></returns>
+        [HttpPut("SystemStatus")]
+        [Authorize(Roles = "Admin")]
+        public ActionResult PutArtistAlleyStatus([FromBody][Required] ArtistAlleySystemStatus status)
```

Comment:
Should be managed via appsettings for now, since it's a last resort kind of feature and storing state inside the container as a file is not an option.